ninja-core/src/site/markdown/documentation/uploading_files.md

```diff
@@ -62,6 +62,60 @@ public Result uploadFinish(Context context) throws Exception {
 }
 </pre>
 
+New way of handling file uploads
+--------------------------------
+Would it be nice if uploaded files of a multipart request could be injected
+directly into controller method just like path or query parameters?
+Ninja allows you to do so -- to inject uploaded file `@FileParam` annotation
+is used. Extracted type of this annotation is `NinjaFileItemStream` from which
+you can get `java.io.InputStream` to read file contents from. There is also a
```

Comment:
Has to be updated to reflect the new NinjaFileItemStream annotations and usage...